src/CaptainsLog/scripts/Utils.java

```diff
@@ -119,6 +122,10 @@ public static int tryCreateUnsearchedRuinsReports(List<SectorEntityToken> entiti
         int count = 0;
 
         for (SectorEntityToken entity : entities) {
+            MarketAPI market = entity.getMarket();
+            if (market == null || market.getSurveyLevel() != MarketAPI.SurveyLevel.FULL) {
```

Comment:
Use old logic for catch-up (adding mod mid-game) - only fully surveyed will have an intel added.